bin/helpers/load_environment.sh

```diff
@@ -5,13 +5,3 @@ if [ ! -f .env ]; then
     exit 1
 fi
 source .env
-
-COMMIT="$TRAVIS_COMMIT"
```

Comment:
Should we move these to .evn_example maybe?

Reply:
Well they are optional and travis specific anyway. Should we trash that .env file even more?